moredata/utils/osm_downloader.py

```diff
@@ -53,10 +53,10 @@ def get_places(self, place_name, key, value, query=None, tags=("name","geom")):
             properties = data["properties"]
             if "tags" in properties.keys():
                 tags = properties["tags"]
-
```

Comment:
Here, to create key and value properties in `json`, you can assign `properties['value'] = value` and `properties['key'] = key`.